omod/src/main/webapp/dictionary/concept.jsp

```diff
@@ -224,13 +224,16 @@
 		<c:if test="${command.concept.datatype != null && command.concept.datatype.name == 'Coded'}">
 			<tr class="localeSpecific">
 				<th valign="top"><openmrs:message code="Concept.answers"/></th>
-                <c:forEach items="${command.locales}" var="loc">
-                    <td class="${loc}">
-                        <c:forEach items="${command.conceptAnswersByLocale[loc]}" var="answer">
-                            <a href="concept.htm?conceptId=${fn:substring(answer.key, 0, fn:indexOf(answer.key, '^'))}"><c:out value="${answer.value}" /> (${fn:substring(answer.key, 0, fn:indexOf(answer.key, '^'))})</a><br/>
-                        </c:forEach>
-                    </td>
-                </c:forEach>
+				<c:forEach items="${command.locales}" var="locale">
+					<td class="${fn:escapeXml(locale)}">
```

Comment:
Was escaping the locale also part of the ticket?

Reply:
Yes,For example, if `conceptId` was `1<2`, it would be escaped to `1&lt;2`, which prevents the browser from interpreting it as HTML. This is a common practice when inserting user-provided data into HTML to prevent XSS attacks.

Reply:
This has been removed, I will have a separate ticket for this change. This is important for prevent XSS attacks